cmd/helpers.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+    "github.com/anaskhan96/soup"
+	"os/exec"
+    "runtime"
+)
+
+// fetchHTML - fetches and parses HTML from the given URL
+func fetchHTML(url string) string {
+    doc, err := soup.Get(url)
+    if err != nil {
+        fmt.Println("Error fetching the URL. Make sure you're connected to Amrita WiFi or VPN.")
+        return ""
+    }
+    return doc
+}
+
+
+// openBrowser - opens a URL in the default web browser
```

Comment:
Hiphens not needed. Just adds complexity. Better to use function name as first word in comment.
nit: End comments with full-stop.
```suggestion
// openBrowser opens a URL in the default web browser.
```